release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -76,11 +76,134 @@ The {product-title} installation program for Google Cloud Platform (GCP) now cre
+
+* HyperPAV enabled on IBM System Z /LinuxONE for the virtual machines for FICON attached ECKD storage
+
+[discrete]
+===== Restrictions
+
+Note the following restrictions for {product-title} on IBM Z and LinuxONE:
+
+* {product-title} for IBM Z does not include the following Technology Preview features:
+** Precision Time Protocol (PTP) hardware
+
+* The following {product-title} features are unsupported:
+** Automatic repair of damaged machines with machine health checking
+** CodeReady Containers (CRC)
+** Controlling overcommit and managing container density on nodes
+** CSI volume cloning
+** CSI volume snapshots
+** FIPS cryptography
+** Helm command-line interface (CLI) tool
+** Multus CNI plug-in
+** NVMe
+** OpenShift Metering
+** OpenShift Virtualization
+** Tang mode disk encryption during {product-title} deployment
+
+* Worker nodes must run {op-system-first}.
+* Persistent shared storage must be provisioned by using either NFS or other supported storage protocols
+* Persistent non-shared storage must be provisioned using local storage, like iSCSI, FC, or using LSO with DASD, FCP, or EDEV/FBA.
+
+[id="ocp-4-8-ibm-power"]
+==== IBM Power Systems
+
+With this release, IBM Power Systems are now compatible with {product-title} {product-version}.
+
+* xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on IBM Power Systems]
+* xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on IBM Power Systems in a restricted network] for installation instructions.
+
+[discrete]
+===== Notable Enhancements
+
+The following new features are supported on IBM Power Systems with {product-title} {product-version}:
+
+* Log forwarding
+* Encrypting data stored in etcd
+* OpenShift Pipelines
+* OpenShift Serverless
+* 3-node cluster support
+* Multus SR-IOV
+
+[discrete]
+===== Supported Features
+
+The following features are also supported on IBM Power Systems:
+
+* Currently, four Operators are supported:
+** Cluster-Logging-Operator
+** Cluster-NFD-Operator
+** Elastic Search-Operator
+** Local Storage Operator
+
+* CodeReady Workspaces
+* Developer CLI - odo
+* HostPath
+* Multipathing
+* Persistent storage using iSCSI
+* OpenShift Service Mesh
+* OVN-Kubernetes with an initial installation of {product-title} 4.8
+* Persistent storage using Fibre Channel
+* Persistent storage using Raw Block
+* 4K Disk Support
+
+[discrete]
+===== Restrictions
+
+Note the following restrictions for {product-title} on IBM Power Systems:
+
+* The following {product-title} features are unsupported:
+** CodeReady Containers (CRC)
+** OpenShift Metering
+** OpenShift Virtualization
+
+* FIPS cryptography
+* Persistent storage must be of the Filesystem type that uses local volumes, Network File System (NFS), or Container Storage Interface (CSI)
```

Comment:
Can you add OpenShift Container Storage here?

Reply:
@ktania46 I've verified with Thomas Stober please add OCS to the notable enhancements for both Power and Z. 

I'm not sure if we should already say:  Red Hat OpenShift Data Foundation 4.7  or OpenShift Container Storage 4.7.

Reply:
@ktania46 I found an entry for OCS in the common attributes I would use that for the entries. 

`:rh-storage-first:`

I assume the attribute gets changed once the OCS docs themselves have implemented the new name. 
Correct @vikram-redhat?

Reply:
Yeah OCS should change it first. Do you have the contact for the OCS docs team?

Reply:
Yes its Anjana Sriram I can reach out to her and ask to inform us once they've implemented the name change.